packages/vaadin-grid/src/vaadin-grid-row-details-mixin.js

```diff
@@ -102,23 +101,28 @@ export const RowDetailsMixin = (superClass) =>
             delete row.querySelector('[part~=details-cell]')._instance;
           });
         }
-
-        if (this.detailsOpenedItems.length) {
```

Comment:
Seems that it is a redundant call since we call `_updateItem` in the `_detailsOpenedItemsChanged` observer. 

Removing this code doesn't change anything and all the tests pass.